packages/xo-web/src/xo-app/user/index.js

```diff
@@ -436,6 +436,7 @@ export default class User extends Component {
                 <option value='zh'>简体中文</option>
                 <option value='hu'>Magyar</option>
                 <option value='tr'>Türkçe</option>
+                <option value='it'>Italiano</option>
```

Comment:
Alphabetical (Unicode) order maybe?

Even if that's not ideal for non-English-like strings, it's, IMHO, better than nothing.

I'll understand if you prefer to do it in another PR though.